sdcm/nemesis.py

```diff
@@ -462,6 +464,24 @@ def disrupt_nodetool_decommission(self, add_node=True):
                 if new_node:
                     new_node.running_nemesis = None
 
+    def disrupt_nodetool_seed_decommission(self, add_node=True):
+        if len(self.cluster.seed_nodes) < 2:
```

Comment:
@roydahan WDYT, on this, i.e. we going on the safe side, and applying this only when more then 2 seed nodes are configured ?
or we should go even when there's one ? since it would break the test quickly anyhow if this one isn't finished ?

Reply:
Yes, that’s fine.
Only thing is to add back a seed node after decommissioning one.